### PR TITLE
Beta release mini update (CUReviews and Requirements)

### PR DESCRIPTION
**Pull request by willespencer**
### Summary 

This pull request updates CUReviews data (as it is currently broken on courseplan.io) and adds a few small requirements updates.

- [x] Update CUReviews fetch to handle new JSON structure
- [x] Add urban planning major and a few more minors
- [x] A few small reqs fixes (CALS, Foreign Lang, and minors listed by college) 

### Test Plan 

Ensure that CUReviews data properly displays and that new major/minors work as expected.

### CODE REVIEW

**Comment by einc** [comment]
> Looks good- just I’m not sure I understand why the FL requirement was deleted from A&S and the math minor

There were duplicate FL requirements haha so I removed one! I also sorted the minors alphabetically so it's not deleted (just moved lower haha)

File: src/components/Requirements.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@
           <div :style="{'border-bottom': minor.display ? `2px solid #${reqGroupColorMap[req.group][0]}` : ''}"
             @click="activateMinor(id)" class="major-title" v-for="(minor, id) in minors" :key="minor.id">
             <p :style="{'font-weight': minor.display ? '500' : '', 'color' : minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}"  class="minor-title-top">{{minor.minorFN}}</p>
-            <p :style="{'color': minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}" class="minor-title-bottom">({{user.collegeFN}})</p> <!-- Change for multiple colleges -->
```
Comment (hanywang2) [inline src/components/Requirements.vue:24]: What was this for again?

Comment (willespencer) [reply src/components/Requirements.vue:24]: @einc explained it in #118 - but it used to display the name of the college under the minor, even though minors are not dependent on what college they are in. Hiding it for now.

Comment (hanywang2) [reply src/components/Requirements.vue:24]: Gotcha

**Pull request merged**